src/api/Nhs.Appointments.Core/AvailabilityCalculator.cs

```diff
@@ -22,9 +22,11 @@ public async Task<IEnumerable<SessionInstance>> CalculateAvailability(string sit
 
             var bookings = await bookingDocumentStore.GetInDateRangeAsync(from.ToDateTime(new TimeOnly(0, 0)), until.ToDateTime(new TimeOnly(23, 59)), site);
 
-            var isNotCancelled = (Booking b) => b.Outcome?.ToLower() != "cancelled";
-            var isNotExpiredProvisional = (Booking b) => b.Provisional == false || b.Created.AddMinutes(5) > time.GetUtcNow();
-            var liveBookings = bookings.Where(isNotCancelled).Where(isNotExpiredProvisional);
+            var liveStatuses = new[] { AppointmentStatus.Booked, AppointmentStatus.Provisional };
```

Comment:
This is probably just because I've not had my head in the bookings code as much, so feel free to ignore me if all this makes sense, but I just wanted to query whether "live" was the right term here. It feels odd to me that a provisional booking would be considered live

Reply:
I think a provisional booking is potentially live as it is affecting availability. Happy to adopt a different name, but nothing better springs to mind